mod/node-api/engines/echo/vaildator.go

```diff
@@ -79,19 +80,19 @@ func ConstructValidator() *validator.Validate {
 
 func ValidateStateID(fl validator.FieldLevel) bool {
 	allowedValues := map[string]bool{
-		"head":      true,
-		"genesis":   true,
-		"finalized": true,
-		"justified": true,
+		utils.StateIDHead:      true,
```

Comment:
probably should have a constants folder

Reply:
create a constants file within the echo package, similar to how it is in other packages.